src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -52,6 +52,9 @@ internal static class FeatureOnOffOptions
         public static readonly PerLanguageOption2<bool> RenameTrackingPreview = new(nameof(FeatureOnOffOptions), nameof(RenameTrackingPreview), defaultValue: true,
             storageLocations: new RoamingProfileStorageLocation(language => language == LanguageNames.VisualBasic ? "TextEditor.%LANGUAGE%.Specific.RenameTrackingPreview" : "TextEditor.%LANGUAGE%.Specific.Rename Tracking Preview"));
 
+        public static readonly PerLanguageOption2<bool> InlineErrors = new PerLanguageOption2<bool>(nameof(FeatureOnOffOptions), nameof(InlineErrors), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.InlineErrors"));
```

Comment:
i thought we already hve an option for this above.  why do we need this guy?

Reply:
It should've been removed, but it was forgotten. Removed.